validator/packaging/centos/rpm-install-script

```diff
@@ -1,3 +1,17 @@
+# Copyright 2018 Intel Corporation
```

Comment:
Is Intel claiming copyright of a single-line script? ;)

I suspect this centos file doesn't work anymore and could be purged.